packages/gestalt/src/TypeaheadInputField.js

```diff
@@ -170,7 +170,9 @@ const TypeaheadInputFieldWithForwardRef: React$AbstractComponent<
             right
           >
             <Icon
-              accessibilityLabel=""
+              accessibilityLabel={
+                !hasValue ? 'Show options' : 'Clear selection'
```

Comment:
@ayeshakmaz we can not have any English strings in any Gestalt component since they would not be internationalized.

Instead we add extra accessibility labels, e.g. `accessibilityModalLabel`

Reply:
Removed this for now, and will address in[ PDS-2337](https://jira.pinadmin.com/browse/PDS-2337)